shaders/src/getters.vs

```diff
@@ -22,44 +22,31 @@ mat3 getWorldFromModelNormalMatrix() {
 
 #if defined(HAS_SKINNING_OR_MORPHING)
 vec3 mulBoneNormal(vec3 n, uint i) {
-    vec4 q  = bonesUniforms.bones[i + 0u];
-    vec3 is = bonesUniforms.bones[i + 3u].xyz;
-
-    // apply the inverse of the non-uniform scales
-    n *= is;
-    // apply the rigid transform (valid only for unit quaternions)
-    n += 2.0 * cross(q.xyz, cross(q.xyz, n) + q.w * n);
-
-    return n;
+    highp mat3 transform = mat3(
+        bonesUniforms.transform[i][0].xyz,
+        bonesUniforms.transform[i][1].xyz,
+        bonesUniforms.transform[i][2].xyz
+    );
+    return normalize(cofactor(transform) * n);
```

Comment:
Should we even normalize here if we normalize in the fragment shader?

Reply:
yes, unfortunately, because of that:
```
void skinNormal(inout vec3 n, const uvec4 ids, const vec4 weights) {
    n =   mulBoneNormal(n, ids.x) * weights.x
        + mulBoneNormal(n, ids.y) * weights.y
        + mulBoneNormal(n, ids.z) * weights.z
        + mulBoneNormal(n, ids.w) * weights.w;
}
```